.github/workflows/integrationTest.yml

```diff
@@ -1,799 +0,0 @@
-# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
-# SPDX-License-Identifier: MIT
-
-name: Run Integration Tests
```

Comment:
This file was never in this repo. Why is it marked deleted?

Reply:
Hmm !!! I believe it is there though !!!! https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/.github/workflows/integrationTest.yml

Reply:
I see it in the repo.